index.html

```diff
@@ -58,7 +60,221 @@ <h3>
 
 </section>
    
+
+<header class="title-header">
+    <h2 class="title">My Recent Works</h2>
+  </header>
+  <section class="recent-works">
+      <div class="work">
+          <section >
+  <img src="/img/Img-Placeholder.png" alt="">
+          </section>
+          <div class="mg">
+  
+        
+          <section class="stories">
+             Multi-Post Stories
+          </section>
+          <section class="story-details">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          </section>
+          <section class="langs">
+              <div>css</div>
+              <div>html</div>
+              <div>bootstrap</div>
+              <div>ruby</div>
+          </section>
+          <section class="disp-btn">
+              <button class="pst-btn">see project</button>
+          </section>
+      </div>
+      </div>
+      <div class="work-2">
+          <div class="proj-work">
+
+         
+          <section class="projects">
+              Profesional Art Printing Data
+           </section>
+           <section class="projects-details">
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+           </section>
+           <section class="langs-2">
+               <div>html</div>
+               <div>bootstrap</div>
+               <div>ruby</div>
+           </section>
+           <section class="proj-btn">
+               <button class="see">see project</button>
+           </section>
+        </div>
+      </div>
```

Comment:
- [ ] In all your projects your missing : 
- The img tag for the project image.
- All the text is not showing.
![image](https://user-images.githubusercontent.com/43698511/129449569-b4064b2b-0ace-40d2-af28-62a22a0fa0a8.png)
Kindly fix the issues in all the project divs.